### PR TITLE
Merge some changes to the state machine

### PR DESCRIPTION
**Pull request by maxh65000**
We tried to get drive practice working tonight but I had to make some changes to the state machine and shooter code.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 1]
```diff
@@ -97,17 +98,19 @@ public void target() {
...
   /** Stops the roller. */
   public void stop() {
+    System.out.println("Stopping roller");
     mRoller.setSpeed(0);
   }
 
   /** Returns whether roller is at full speed. */
   public boolean isAtVelocity() {
-    return mRollerVelocity > TARGET_ROLLER_VELOCITY;
+    return true;// mRollerVelocity > TARGET_ROLLER_VELOCITY;
```
Comment (codemankey) [inline src/main/java/frc/robot/subsystems/Shooter.java:114]: Why do we want this to always return true now?

Comment (maxh65000) [reply src/main/java/frc/robot/subsystems/Shooter.java:114]: https://github.com/Team138Entropy/Entropy2020/issues/99

**Review by codemankey — APPROVED** [review]
Got the answer to my question in discord from Max and Liam. A separate ticket has been opened to address the actual issue.

**Comment by maxh65000** [comment]
See also #99

**Pull request merged**